### ISSUE TITLE
Permit labels in function calls

### ISSUE THREAD

**Issue by lpil** [michaeljones/gleam-templates#18]
I think permitting labels would make code clearer. Thank you!
```
Unexpected identifier: oob:

error: 
   ┌─ ./src/todomvc/web/templates/home.gleamx:68:49
   │
68 │           {[ item_template.render_builder(item, oob: False) ]}
   │  
```

**Comment by michaeljones** [comment]
Hopefully should be fixed in 0.11.0. 

I realise you might have found other approaches to this by now.

**Comment by lpil** [comment]
Thank you!
